jax/scipy/linalg.py

```diff
@@ -264,3 +264,114 @@ def tril(m, k=0):
 @_wraps(scipy.linalg.triu)
 def triu(m, k=0):
   return np.triu(m, k)
+
+@_wraps(scipy.linalg.expm)
+def expm(A, isuppertriangular=False):
```

Comment:
A few thoughts here:
- Naming: Maybe just `upper_triangular`? I find `isuppertriangular` a little hard to read.
- Backwards compatibility: To avoid potential conflicts with SciPy (if/when they add their own new keyword arguments into `expm`), let's use a keyword only argument here, i.e., `def expm(A, *, isuppertriangular=False):`
- Documentation: You'll need to add JAX specific documentation on this argument (use `lax_description`), because it isn't found in SciPy.

Reply:
Thanks for the great suggestions. I did not consider backwards compatibility. I have made these changes. 
For the documentation, I am not sure about how to add a parameter description that will look exactly like the original SciPy documentation. I looked at other lax_description strings to come up with what has been committed.